internal/controller/operatorconfigmap_controller.go

```diff
@@ -500,6 +500,39 @@ func (c *OperatorConfigMapReconciler) shouldEnableDriver(driverKey string) bool
 	return enableDriver
 }
 
+// getTopologyLabels returns a map of topology labels from the storage clients status and if the
+// storage clients status does not have any topology labels, it uses the configmap default values.
+func (c *OperatorConfigMapReconciler) getTopologyLabels(storageClients *v1alpha1.StorageClientList) map[string]struct{} {
+	topologyDomainLablesSet := map[string]struct{}{}
+
+	// First, merge topology keys from StorageClient Status
+	// Multiple storage clients hubs can have different topology keys
+	for i := range storageClients.Items {
+		storageClient := &storageClients.Items[i]
+		if storageClient.Status.RbdDriverRequirements != nil {
+			// if the topology domain labels value is not empty, it should be a list of labels
+			// e.g. "region,zone"
+			for _, label := range storageClient.Status.RbdDriverRequirements.TopologyDomainLabels {
+				topologyDomainLablesSet[label] = struct{}{}
+			}
+		}
+	}
+
+	// If no topology labels from StorageClient status, use ConfigMap defaults
+	if len(topologyDomainLablesSet) == 0 {
```

Comment:
we should not merge it? in case for multi cluster

Reply:
We'll not merge if there is topology label in storageClient status